src/main/resources/assets/jei/lang/de_DE.lang

```diff
@@ -1,64 +1,88 @@
 # Tooltips
-jei.tooltip.config=Konfiguration
+jei.tooltip.config=JEI-Konfiguration
+jei.tooltip.show.recipes=Zeige Rezepte
 jei.tooltip.show.all.recipes=Zeige alle Rezepte
 jei.tooltip.delete.item=Klicke zum Löschen
 jei.tooltip.liquid.amount.with.capacity=%,d / %,d mB
 jei.tooltip.liquid.amount=%,d mB
+jei.tooltip.transfer=Bewege Gegenstände
+jei.tooltip.recipe.ore.dict=Akzeptiert: %s
+jei.tooltip.item.colors=Farben: %s
+jei.tooltip.shapeless.recipe=Formloses Rezept
+jei.tooltip.cheat.mode=Cheat-Modus
 
 # Error Tooltips
-jei.tooltip.error.recipe.transfer.missing=Fehlende Items
-jei.tooltip.error.recipe.transfer.inventory.full=Inventar ist voll
+jei.tooltip.error.recipe.transfer.missing=Fehlende Gegenstände
```

Comment:
Two words meaning the same thing is a concept that exists and is used even within a language. Your "false consistency" is just your inflexibility in accepting that.

I suggest the following reasonable rule:
_If there's not enough space in the gui and the programmers are not willing or able to change it, than the word must not be translated. To avoid to be forced to not translating a word everewhere else based on the false concept of error reproducing consistency, the translated and the not translated word are considered as synonyms. Therefore it is ok to have both within the same game/mod._
This would work because the people know the meaning of both words (as you've confirmed). Even you would be able to live with it, just be a little bit flexible instead of forcing that only one word is allowed.

So we're again at the point where we were 4 days ago:
I still would prefer to use "Gegenstand" within complete sentences where enough space is available, like helping tooltips and so on, but something like a "Itemliste" would be ok. What do you think of this "hybrid mode"?

Can you allow synonyms or must it be one word no matter what?